browser/android/youtube_script_injector/youtube_script_injector_tab_helper.cc

```diff
@@ -317,4 +331,11 @@ void YouTubeScriptInjectorTabHelper::SetFullscreenRequested(bool requested) {
   }
 }
 
+void YouTubeScriptInjectorTabHelper::OnFullscreenScriptComplete(
+    content::GlobalRenderFrameHostToken token,
+    base::Value value) {
+  // Reset fullscreen state when the script completes
```

Comment:
need to check here that `token == web_contents()->GetPrimaryMainFrame()->GetGlobalFrameToken`